script.js

```diff
@@ -1,4 +1,114 @@
+// We first need to get the global variables
+
+// game modes
+var gamemodeRollDice = "game mode dice roll";
+var gamemodeChooseDice = "game mode choose dice";
+var gamemodeChooseWinner = "game mode choose winner";
```

Comment:
since "game mode" is constant throughout the 3 global variables, removing "game mode" is fine